src/index.js

```diff
@@ -11,7 +11,4 @@ ReactDOM.render(
   document.getElementById('root'),
 );
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
 reportWebVitals();
```

Comment:
- [ ] Delete all the boilerplate that was created that you will not need in your app